src/AuthenticationPortal.Services/Services/AwsCognito.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Amazon.CognitoIdentityProvider;
+using Amazon.CognitoIdentityProvider.Model;
+using Amazon.Extensions.CognitoAuthentication;
+using AuthenticationPortal.Contracts;
+using Microsoft.Extensions.Options;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace AuthenticationPortal.Services
+{
+    public class AwsCognito : IUserAuthenticationAdapter
+    {
+        private readonly AwsCognitoCredentials _settings;
+        public AwsCognito(IOptions<AwsCognitoCredentials> settings)
+        {
+            _settings = settings.Value;
+        }
+
+        static Amazon.RegionEndpoint region = Amazon.RegionEndpoint.APSouth1;
+        private SignInResponse _signInResponse = new SignInResponse();
```

Comment:
this should be part of the method which is returning the response.

Reply:
Done